test/sql/updates/test-0.1.1.sql

```diff
@@ -1,4 +1,8 @@
-\d+ _timescaledb_catalog.*;
+\dt+ _timescaledb_catalog.*;
+\di+ _timescaledb_catalog.*
+-- Do not list sequence details because of potentially different state
```

Comment:
Can't we just restart the two containers to make the sequence details the same?

Reply:
The big problem I see here is that without a `\d+ _timescaledb_catalog.*` then we don't output the catalog tables' columns or indexes. This, I think is critical. If we want to fix this I think we need to `\d+ _timescaledb_catalog.hypertable` for every table in our catalog.

Reply:
Any suggestion on how to get around this? Listing sequences won't work, because their state will be different in updated and restored.